v2/thirdparty/custom-ui/thirdparty-login.mdx

```diff
@@ -19,11 +19,15 @@ import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
 
 There are different flows that the third party login provider may support:
 - _**Flow 1)**_ Via authorisation code (one time use code) - for web and mobile apps. 
-    - _**a)**_ If you have configred the client secret on the backend:
+    - _**a)**_ If you have configured the client secret on the backend:
         - The frontend sends the auth code to the backend, the backend exchanges that with the provided client secret to get the access token. The access token is then used to fetch user info and log them in.
     - _**b)**_ If you have **not** provided the client secret on the backend:
         - The backend uses PKCE flow to exchange the auth code with the user's access token. The access token is then used to fetch user info and log them in.
-- _**Flow 2)**_ Via OAuth / access tokens - for mobile apps.
+    - _**c)**_ If you have configured the client secret on the backend and also need PKCE for additional security:
```

Comment:
no need to have this mode of client secret + pkce. Just have client secret or just pkce